test/db/cmd/metadata

```diff
@@ -653,27 +653,18 @@ s 0x100
 Cd 4 1
 C*~Cd
 ?e --1--
-Cd- 4 1
-C*~Cd
-?e --2--
 Cd 4 1
 Cd-
 C*~Cd
-?e --3--
+?e --2--
 Cd 4 1
 C-
 C*~Cd
-?e --4--
-Cd 4 1
-C- 4
-C*~Cd
```

Comment:
why?

Reply:
I see no reason to have meaningless `C- 4` command that removes the metadata. The removal function doesn't need to know the size of the metadata IMHO, to remove it.